uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/test/shared/controllers/sub-controllers/request-modal-details-toil.controller.spec.js

```diff
@@ -302,6 +320,34 @@ define([
           expect(controller.uiOptions.toil_duration_in_hours).toBe(2);
         });
       });
+
+      describe('when user opens TOIL accrual options group editor', function () {
+        beforeEach(function () {
+          // flushing TOIL accrual options
+          controller.toilAmounts = null;
+
+          spyOn(crmAngService, 'loadForm').and.callThrough();
+          controller.openToilInDaysAccrualOptionsEditor();
+        });
+
+        it('calls the CRM.loadForm with according URL', function () {
+          expect(crmAngService.loadForm).toHaveBeenCalledWith('/civicrm/admin/options/hrleaveandabsences_toil_amounts?reset=1');
+        });
+
+        describe('when TOIL accruals are edited', function () {
+          beforeEach(function () {
+            $rootScope.$digest();
+          });
+
+          it('reloads TOIL accrual options', function () {
+            expect(Object.keys(controller.toilAmounts).length).toBeGreaterThan(0);
+          });
+
+          it('does not cache TOIL accrual options', function () {
```

Comment:
> "does not cache"

This is misleading, because based on your changes the data is always cached, we're just flushing the cache before caching the data again

Reply:
Done.